code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -42,98 +45,131 @@
 				playsound(user,'sound/weapons/egun_toggle_noammo.ogg',73,0,-5)
 	update_icon()
 
+////////////////////////////ADVANCED ENERGY GUN////////////////////////////////////////////////////
+
 /obj/item/weapon/gun/energy/gun/nuclear
-	name = "Advanced Energy Gun"
-	desc = "An energy gun with an experimental miniaturized reactor."
+	name = "\improper Advanced Energy Gun"
```

Comment:
no real reason this shouldn't just be converted to lowercase entirely

Reply:
I kinda like the capitalization, and it's how it's always been. This way at least now we get "the".